Rakefile

```diff
@@ -43,7 +43,27 @@ task :build_template do
   builder = TemplateBuilder.new(File.join(__dir__, "app/templates/install"))
   contents = File.read(File.join(__dir__, "app/templates/install/template.rb"))
 
-  puts ERB.new(contents).result(builder.get_binding)
+  ERB.new(contents).result(builder.get_binding)
```

Comment:
Why we removed the `put`, btw?

Reply:
I suggested that we don't need it since we changed a way to publish template.

Reply:
It's still useful to debug the template locally (I usually do smth like `be rake build_template > tmp/template.rb`)